### DISCUSSION TITLE
2023년 08월 30일

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#77] [General]
# 2023년 08월 30일

LOG 레벨 인터뷰가 있었다. CORS 를 준비를 많이 못해가서 그 부분 답변이 미흡했다. 그 외에는 답변을 잘 한 것 같은데, 좀 더 기술 쪽으로 딥한 질문이 나왔다면 좋았을 것 같다. 배운 것 [ cors ] 를 정리했다. 느낀 것

https://vero.wiki/2023-08-30/

**Comment by carsago** [comment]
아 나도 cors 궁금한데

**Reply by Cyma-s** [reply]
같이 공부하시죠
